backend/app/controllers/api/v1/answers_surveys_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::AnswersSurveysController < Api::V1::ApiController
+  def create
+    answers_survey = AnswersSurvey.new(answers_surveys_params.merge(user: current_user))
```

Comment:
I think it is better to have `.merge(user: current_user)` inside `answers_surveys_params` method, because this way we have all params coming from one place

Reply:
Oh didn't know that was possible!